packages/smartcontracts/src/tests/testUtils/deployment.ts

```diff
@@ -20,6 +20,8 @@ export async function deployContracts(): Promise<BridgeDeploymentResult> {
     accounts[1],
     // relayer address
     accounts[0],
+    // community wallet address
+    accounts[4],
```

Comment:
```suggestion
    accounts[3],
```

I would say be consistent with `deployBridgeProxy` and use accounts[3]

Reply:
accounts[3] is being used in [PR](https://github.com/WavesHQ/bridge/pull/323) [here](https://github.com/WavesHQ/bridge/blob/520b98f1cb2591d6ff28f32feeb9112b3472d395/packages/smartcontracts/src/tests/testUtils/deployment.ts#L26). Trying to have them separate to avoid any failing test after the merge.